input/postgres/relation_column_stats.go

```diff
@@ -20,9 +21,21 @@ func GetColumnStats(ctx context.Context, logger *util.Logger, db *sql.DB, global
 	var sourceTable string
 
 	if StatsHelperExists(ctx, db, "get_column_stats") {
-		logger.PrintVerbose("Found pganalyze.get_column_stats() stats helper")
-		sourceTable = "pganalyze.get_column_stats()"
-		server.SelfTest.MarkDbCollectionAspectOk(dbName, state.CollectionAspectColumnStats)
+		if strings.Contains(StatsHelperReturnType(ctx, db, "get_column_stats"), "pg_stats") {
```

Comment:
This introduces an additional DB round trip for every `GetColumnStats` call. Since we only do that every full snapshot, though, I doubt it can possibly matter.